src/components/inputLabel.jsx

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React, { type ElementProps } from "react";
+import styled from "styled-components";
+import Input from "./inputs";
+
+type lableObject = ElementProps<typeof Input> & {
+  id: string,
+  label: string,
+  error?: string
+};
+
+const Lable = styled.label`
+  display: inline-block;
+  font-size: 20px;
+  font-weight: 400;
+  color: #212529;
+`;
+
+const LableError = styled.div`
+  color: #721c24;
+  background-color: #f8d7da;
+  border-color: #f5c6cb;
+  padding: 8px 20px;
+  border-bottom-left-radius: 5px;
+  border-bottom-right-radius: 5px;
+`;
+
+const InputLabel = ({ id, label, error = "", ...rest }: lableObject) => (
```

Comment:
The '...rest' is the basic attributes of input.  I don't know what input attributes will be used. But I can remove it, because I only use the 'type' attribute in the '...rest'. Remove?